pkg/ddl/index.go

```diff
@@ -89,6 +89,9 @@ var (
 )
 
 func suppressErrorTooLongKeyKey(sctx sessionctx.Context) bool {
```

Comment:
I renamed it to suppressErrorTooLongKeyForSchemaTracker, because we use other ways to suppress other error. For example, set ClusteredIndexDefModeOff to allow drop PK. Maybe in future we can unify all DM's special suppressing to this function.